### ISSUE TITLE
Navigator Error

### ISSUE THREAD

**Issue by KEAppleby** [ContinuumIO/anaconda-issues#4174]
## Main error
An unexpected error occurred on Navigator start-up<br>psutil.AccessDenied (pid=14468)
## Traceback
```
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 620, in wrapper
    return fun(self, *args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 690, in cmdline
    ret = cext.proc_cmdline(self.pid)
PermissionError: [WinError 5] Access is denied

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\app\start.py", line 108, in start_app
    if misc.load_pid() is None:  # A stale lock might be around
  File "C:\ProgramData\Anaconda3\lib\site-packages\anaconda_navigator\utils\misc.py", line 384, in load_pid
    cmds = process.cmdline()
  File "C:\ProgramData\Anaconda3\lib\site-packages\psutil\__init__.py", line 701, in cmdline
    return self._proc.cmdline()
  File "C:\ProgramData\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 623, in wrapper
    raise AccessDenied(self.pid, self._name)
psutil.AccessDenied: psutil.AccessDenied (pid=14468)

```
## System information
```
python: 3.6.1
language: en_US
os: Windows;10;10.0.15063;AMD64;Intel64 Family 6 Model 60 Stepping 3, GenuineIntel
version: 1.6.2
platform: win-32
qt: 5.6.2
pyqt: 5.6.0
conda: 4.3.21
```

**Comment by goanpeca** [comment]
**See Issue #1984 for more information on how to fix this.**

---

Closing as duplicate of #1984


---

Please remember to update to the latest version of Navigator to include
the latest fixes.

Open a terminal (on Linux or Mac) or the Anaconda Command Prompt (on windows)
and type:

```
$ conda update anaconda-navigator
```

**Comment by KEAppleby** [comment]
Thank you.


I expect I got into this mess partly because I tried to update to the newest version of Anaconda yesterday.  I kept getting a message that I didn't have the permissions needed to write in the Anaconda directory.  I tried all sorts of things to give all sorts of user groups access to the directory, tried to make sure inheritances were all set correctly but never got to where I could get write permission to C:/Programdata/Anaconda3 (I am using an administrative account; I am the system administrator).  The error  message from Anaconda said I could clone the directory in a directory I own, which I tried, which then made versions of each Anaconda product (Navigator, prompt, cloud etc.) with an xlg designation (my username) pop up.  But I still couldn't get write permission.  I tried uninstalling and re-installing and that didn't help anything.


After resetting Anaconda per your instructions, I still got the message about not having permission to write in the directory, so that has sent me back to where I was yesterday.  However, I tried upgrading using Navigator through the xlg environment, and that seems to have worked.


Fingers crossed!  Thanks again.


Karen Appleby



________________________________
